api/src/main/java/gov/cms/ab2d/api/controller/common/ApiCommon.java

```diff
@@ -87,14 +81,27 @@ public void checkSinceTime(OffsetDateTime date) {
         if (date.isAfter(OffsetDateTime.now())) {
             throw new InvalidClientInputException("You can not use a time after the current time for _since");
         }
-        try {
-            OffsetDateTime ed = OffsetDateTime.parse(SINCE_EARLIEST_DATE, ISO_DATE_TIME);
```

Comment:
I added new constant SINCE_EARLIEST_DATE_TIME with OffsetDateTime type. In this case, we don't need to parse string every time